rllib/utils/test_utils.py

```diff
@@ -1129,3 +1130,36 @@ def check_reproducibilty(
                 results1["info"][LEARNER_INFO][DEFAULT_POLICY_ID]["learner_stats"],
                 results2["info"][LEARNER_INFO][DEFAULT_POLICY_ID]["learner_stats"],
             )
+
+
+def get_cartpole_dataset_reader(batch_size: int = 1) -> "DatasetReader":
```

Comment:
made this in order to remove repetitive code that exists for creating the cartpole dataset.